src/common/util/browser.tsx

```diff
@@ -1,62 +1,68 @@
 import * as React from 'react';
+import * as ReactGA from 'react-ga';
+import { History } from 'history';
 
-let ReactGA, currentLocation;
+declare const window: {
```

Comment:
Basically, I'm declaring to Typescript that the `window` variable is guaranteed to satisfy a certain interface. Otherwise the compiler complains that `window` doesn't have a `location` or `process` field.